ExportExtensios2017/app.config

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <system.serviceModel>
+        <bindings>
+            <wsHttpBinding>
+                <binding name="CustomBinding_IVsIdeService">
+                    <security mode="None" />
+                </binding>
+                <binding name="CustomBinding_IVsIdeService1">
+                    <security mode="Transport">
+                        <transport clientCredentialType="None" />
+                    </security>
+                </binding>
+            </wsHttpBinding>
+        </bindings>
+        <client>
+            <endpoint address="http://co1msdngal97.phx.gbl/Services/dev12/Extension.svc"
```

Comment:
I don't remember well, I think it's something about the extension. I'll be checking it out today.